iam/snippets/snippets.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+var client, _ = google.DefaultClient(
+	context.Background(),
+	iam.CloudPlatformScope)
+var iamService, _ = iam.New(client)
+
+// [START iam_view_grantable_roles]
+func viewGrantableRoles(fullResourceName string) {
```

Comment:
Please add a comment with what this function does, starting with the function name. There should be a newline between the region tag and the comment so the tag doesn't show up in GoDoc.